hardware/arduino/avr/cores/arduino/WInterrupts.c

```diff
@@ -154,26 +234,33 @@ void attachInterrupt(uint8_t interruptNum, void (*userFunc)(void), int mode) {
 
 void detachInterrupt(uint8_t interruptNum) {
   if(interruptNum < EXTERNAL_NUM_INTERRUPTS) {
+    disableInterrupt(interruptNum);
+    intFunc[interruptNum] = 0;
```

Comment:
This should be `nothing` now
http://askubuntu.com/questions/298487/not-enough-free-disk-space-when-upgrading